src/config/dynamic-theme-fixes.config

```diff
@@ -4543,6 +4543,18 @@ body
 
 ================================
 
+discuss.pixls.us
+
+INVERT
+#site-logo.logo-big
+
+CSS
+:root {
+    --secondary-rgb: var(--darkreader-bg) !important;
+}
```

Comment:
So while this is correct... I would like to fix this within the code, because this would currently happen for any discourse instance out their. So if I fix this within the code, every discourse instance would be fixed. If you could leave this out for this PR that would be nice.

Reply:
Sure thing! Please check my work.